### PR TITLE
added new location to Uchicago

### PR DESCRIPTION
**Pull request by ramavavi**
This PR addresses the issue: https://github.com/episphere/connect/issues/757

Changes tracked are available in the box link : https://app.box.com/s/mkipmbpsdz36hgrjuk8iev564gkpr2v0

1. Addition of Orland Park location
2. Removal of days/hours for all locations
3. Added extra styling to the content in the webpage.

### CODE REVIEW

File: js/pages/samples.js [hunk 1]
```diff
@@ -459,19 +459,45 @@ const u_chicago = {
...
+            'Ingalls Memorial Hospital offers free valet and self-parking.',
+        ],
+        [
+            '<span id="uln">UChicago Medicine – South Loop, in Downtown Chicago</span>',
+            'Southgate Market P1 <br>1101 S Canal St </br> Chicago, IL 60607',
+            '',
+            'The clinic is located on the P1 level next to the DSW. Entrances are located off of Canal St right beside the Panera Bread and on the P1 level of the parking garage. We will validate your parking pass from the parking garage. Please show your self-parking ticket to research staff. We are unable to validate street parking.',
+        ],
+        [
+            '<span id="uln">UChicago Medicine – River East, in Downtown Chicago</span>',
```
Comment (Gbarra9) [inline js/pages/samples.js:490]: Change HTML `id` attribute to `class` attribute and provide a descriptive class name value.

File: js/pages/samples.js [hunk 2]
```diff
@@ -459,19 +459,45 @@ const u_chicago = {
...
+            'Located in Lucky Strike Downtown Chicago <br>355 E. Grand Ave <br>Chicago, IL 60611',
+            '',
+            'From Lake Shore Drive: Exit at Grand Ave. and proceed west towards McClurg Ct.  An entrance to the parking garage is located past that intersection on your left at 321 E. Grand.<br><br>From the West: Take Illinois St. east towards Columbus Dr. An entrance to the parking garage is located past that intersection on your left at 300 E. Illinois.<br><br>Once inside the parking garage, follow the signs to LL3 (Fall) for designated UChicago Medicine patient parking.  We will validate your parking pass from the parking garage. Please show your self-parking ticket to research staff. We are unable to validate street parking.',
+        ],
+        [
+            '<span id="uln">UChicago Medicine Orland Park in Orland Park, IL</span>',
```
Comment (Gbarra9) [inline js/pages/samples.js:496]: Change HTML `id` attribute to `class` attribute and provide a descriptive class name value.

File: js/pages/samples.js [hunk 3]
```diff
@@ -588,11 +614,22 @@ const renderLocations = (site) => {
...
+                <div style="width:100%">
+                    <div class="messagesHeaderFont">
+                    </div>
+                    <div class="messagesBodyFont">
+                        ${location[1]}
+                    </div>
+                </div>
+            </div>`
+            }
+            else{
```
Comment (Gbarra9) [inline js/pages/samples.js:628]:
```suggestion
            } else {
```

Formatting suggestion

File: js/pages/samples.js [hunk 4]
```diff
@@ -588,11 +614,22 @@ const renderLocations = (site) => {
...
                 template += `<div class="row" style="width:100%">
                 <div style="width:100%">
                     <div class="messagesHeaderFont">
-                        Address and Directions
+                    Address and Directions
```
Comment (Gbarra9) [inline js/pages/samples.js:632]: Format suggestion: This can be indented.

File: js/pages/samples.js [hunk 5]
```diff
@@ -588,11 +614,22 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
-            } else {
+            } else if(site==u_chicago && !location[1]){
```
Comment (Gbarra9) [inline js/pages/samples.js:617]:
```suggestion
            } else if (site === u_chicago && !location[1]){
```
Format suggestion and strict equality operator

**Review by Gbarra9 — CHANGES_REQUESTED** [review]

File: js/pages/samples.js [hunk 6]
```diff
@@ -588,6 +614,16 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
+            } else if (site === u_chicago && !location[1]){
+                template += `<div class="row" style="width:100%">
+                <div style="width:100%">
+                    <div class="messagesHeaderFont">
```
Comment (anthonypetersen) [inline js/pages/samples.js:620]: should there be something inside of the header div?

**Review by anthonypetersen — COMMENTED** [review]
waiting for answer to question then will approve

File: js/pages/samples.js (hunk 6 above)
Comment (ramavavi) [reply js/pages/samples.js]: Thanks for correcting me Tony, I removed the empty div.

File: js/pages/samples.js [hunk 7]
```diff
@@ -588,6 +614,14 @@ const renderLocations = (site) => {
                         </div>
                     </div>
                 </div>`
+            } else if (site === u_chicago && !location[1]){
+                template += `<div class="row" style="width:100%">
```
Comment (anthonypetersen) [inline js/pages/samples.js:618]: What exactly is this code doing? If `location[1]` is a falsy value, we want to render it?

Comment (ramavavi) [reply js/pages/samples.js:618]: This condition is for the content under the "Where do I donate My Samples" header. To remove the subheading "address and directions" under this header I added that piece of code. But Address and locations will be visible for all the locations.

**Comment by anthonypetersen** [comment]
looks good

**Review by anthonypetersen — APPROVED** [review]

**Comment by ramavavi** [comment]
reviewed dev and looks good. ready for stage.

**Pull request merged**